### DISCUSSION TITLE
migrate database tabel notification

### DISCUSSION THREAD

**Discussion by farhanreizha** [deaafrizal/cuyuniverse-forum#163] [Q&A] [answered]
mau nanya dong kok gua migrate database kok table notification nya ngga muncul ya kenapa dah?? tapi di terminal ke create

![image](https://user-images.githubusercontent.com/48412599/180756371-68285dca-6e16-47cf-9d0d-a2e23c3add90.png)
![image](https://user-images.githubusercontent.com/48412599/180756549-0bb3b1cc-843a-4333-84dc-4d425c87fd84.png)

**Comment by AsfahanX** [comment, marked as answer]
isi nya lagi di [comment](https://github.com/deaaprizal/laract9/commit/983af2ce06a7f332c79f4431000ab9840f225fda) bang
gua juga ketepu. wkwkwk.
mau di improve lagi kyak nya

https://github.com/deaaprizal/laract9/blob/1a7f16c482ce3452d88e355032a78500c7222b76/database/migrations/2022_07_22_195337_create_notifications_table.php#L16-L24

**Reply by farhanreizha** [reply]
oalah pantesan wkwk gua bingung mau ngotak ngatik laravel tapi kaga paham wkwk

**Reply by deaafrizal** [reply]
wkwkwkwk sabar2
